src/components/CheckboxInput/CheckboxInput.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 
-import CheckableInput from "../CheckableInput";
-import type { Props as CheckableInputProps } from "../CheckableInput/CheckableInput";
+import CheckableInput from "./CheckableInput";
+import type { CheckableInputProps } from "./CheckableInput";
```

Comment:
It used to be `import type { Props as CheckableInputProps }`, as I think `CheckableInput` exports `Props`. Does it export `CheckableInputProps`?

Reply:
https://github.com/canonical/react-components/blob/86592fe06a99ea2a473aef3cdb67b7a72e611ca5/src/components/CheckboxInput/CheckableInput/CheckableInput.tsx#L8